### PR TITLE
Covariance samples

### PR DESCRIPTION
**Pull request by Alexandr-Solovev**

### CODE REVIEW

File: samples/oneapi/dpc/ccl/sources/cor_distr_ccl.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+namespace dal = oneapi::dal;
+
+void run(sycl::queue& queue) {
+  const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
+
+  const auto data = dal::read<dal::table>(
+      queue, dal::csv::data_source{data_file_name});
+
+      const auto cov_desc = dal::covariance::descriptor{}.set_result_options(
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/cor_distr_ccl.cpp:40]: Alignment?

File: samples/oneapi/dpc/mpi/sources/cor_distr_mpi.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+namespace dal = oneapi::dal;
+
+void run(sycl::queue& queue) {
+  const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
+
+  const auto data = dal::read<dal::table>(
+      queue, dal::csv::data_source{data_file_name});
+
+      const auto cov_desc = dal::covariance::descriptor{}.set_result_options(
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/mpi/sources/cor_distr_mpi.cpp:40]: Alignment?

File: samples/oneapi/dpc/mpi/sources/cov_distr_mpi.cpp [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            << std::endl;
+  sycl::queue q{device};
+  run(q);
+
+  status = MPI_Finalize();
+  if (status != MPI_SUCCESS) {
+    throw std::runtime_error{"Problem occurred during MPI finalize"};
+  }
+  return 0;
+}
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/mpi/sources/cov_distr_mpi.cpp]: Add new line

File: samples/oneapi/dpc/mpi/sources/cov_distr_mpi.cpp [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+namespace dal = oneapi::dal;
+
+void run(sycl::queue& queue) {
+  const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
+
+  const auto data = dal::read<dal::table>(
+      queue, dal::csv::data_source{data_file_name});
+
+  const auto cov_desc = dal::covariance::descriptor{};
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/mpi/sources/cov_distr_mpi.cpp:40]: Please add result option explicitly

File: samples/oneapi/dpc/ccl/sources/cov_distr_ccl.cpp [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+namespace dal = oneapi::dal;
+
+void run(sycl::queue& queue) {
+  const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
+
+  const auto data = dal::read<dal::table>(
+      queue, dal::csv::data_source{data_file_name});
+
+  const auto cov_desc = dal::covariance::descriptor{};
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/sources/cov_distr_ccl.cpp:40]: Please add result option explicitly

File: samples/oneapi/dpc/ccl/onedal_lnx.lst [hunk 6]
```diff
@@ -18,5 +18,7 @@
 ##     Intel(R) oneAPI Data Analytics Library samples list
 ##******************************************************************************
 
-MPI  =  kmeans_distr_ccl    \
-        dbscan_distr_ccl
+MPI  =  cor_distr_ccl       \
```
Comment (amgrigoriev) [inline samples/oneapi/dpc/ccl/onedal_lnx.lst:21]: Please set in alphabetical order

**Review by makart19 — APPROVED** [review]

**Review by napetrov — APPROVED** [review]

**Pull request merged**